### DISCUSSION TITLE
Task 6 Problem 4

### DISCUSSION THREAD

**Discussion by rockyx50** [chulminy/CIVE497-CIVE700#129] [Q&A] [answered]
In question 4 it states that we must reuse the `computeH `function we wrote in task 5 but then at the bottom says we can use  `fitgeotrans `which does the same thing. Are we allowed to use either function?

**Comment by chulminy** [comment, marked as answer]
I would change the wording:

In addition, you might need your computeH, which was used in Task05.

**Reply by rockyx50** [reply]
okay, thanks!
